src/rest-server/src/models/kubernetes/kubernetes.js

```diff
@@ -99,14 +98,11 @@ const getPods = async (options = {}) => {
   const {namespace} = options;
   const client = getClient();
 
-  const requestOptions = {};
-  requestOptions.params = cloneDeep(options);
-  delete requestOptions.params.namespace;
```

Comment:
do you need to `delete options.namespace`?

Reply:
k8s API server will ignore unknown query parameter. But I don't find any doc for this. Remove `options.namespace` is a more safety way.

I don't want to change the input object, will clone a new object and remote `namespace` property.